### PR TITLE
Lock api

### PR DESCRIPTION
**Pull request by travisleo**
added in the API to set the variables on the lock control api

### CODE REVIEW

File: ece595_project_2/source/lock.c [hunk 1]
```diff
@@ -7,21 +7,23 @@
...
+#define OTHER 2
+#define INSIDE 1
+#define OUTSIDE 0
+
 
 /* Variables */
-static uint8_t Prox_Status; /*TODO WILL COME FROM XBEE*/
-static uint8_t Time_Status; /*TODO WILL COME FROM RTC*/
-static uint8_t Weather_Status; /*TODO WILL COME FROM WEATHER API*/
+static uint8_t Prox_Status;
```
Comment (djaenicke) [inline ece595_project_2/source/lock.c]: These should be the corresponding enum types.

File: ece595_project_2/source/proximity_estimation.c [hunk 2]
```diff
@@ -14,6 +14,7 @@
 #include "assert.h"
 
 #define EXPECTED_RESPONSE_SIZE (22)
+#define THRESHOLD -47
```
Comment (djaenicke) [inline ece595_project_2/source/proximity_estimation.c]: Make this positive

**Review by djaenicke — CHANGES_REQUESTED** [review]
See my comments on the code.

**Comment by travisleo** [comment]
Changes made

**Review by djaenicke — APPROVED** [review]

**Pull request merged**